### PR TITLE
Changes for aucore-blockvote-8 part 3

### PR DESCRIPTION
**Pull request by dt-r**
This PR addresses some but not all issues from the AU Core R1 Ballot for Comment aucore-blockvote-8: https://confluence.hl7.org/display/HAFWG/aucore-blockvote-8

Fixes for the following are in this PR:

- [FHIR-45166](https://jira.hl7.org/browse/FHIR-45166?src=confmacro)
- [FHIR-45159](https://jira.hl7.org/browse/FHIR-45159?src=confmacro)
- [FHIR-45102](https://jira.hl7.org/browse/FHIR-45102?src=confmacro)
- [FHIR-45065](https://jira.hl7.org/browse/FHIR-45065?src=confmacro)
- [FHIR-44957](https://jira.hl7.org/browse/FHIR-44957?src=confmacro)

### CODE REVIEW

**Review by StephThaoOng — COMMENTED** [review]
Fix for FHIR-45159: Line 24: should we update the url link to either:
1. https://sparked.csiro.au/index.php/sparked-products-resources/australian-core-data-for-interoperability/
or,
2. https://sparked.csiro.au/index.php/sparked-products-resources/australian-core-data-for-interoperability/aucdi-release-1/

Currently the url in the input file re-directs to [https://sparked.csiro.au/](https://sparked.csiro.au/)

**Review by StephThaoOng — COMMENTED** [review]
In [Fixes for FHIR-45065 & FHIR-45102](https://github.com/hl7au/au-fhir-core/pull/167/commits/5cab0ca9b37159205233219da00c984dcb008f8e): Line 160: 

> Example: ExplanationOfBenefit resource where the patient’s insurance coverage is not available.
The text has not been updated to align with the code snippet. 
Suggest change to something like "Patient resource where the patient's date of birth is not available".

File: input/pagecontent/general-requirements.md [hunk 1]
```diff
@@ -228,7 +228,7 @@ For mandatory elements (minimum cardinality is > 0), the element **SHALL** be po
 - where a requester may know that the data is suppressed use the code `masked` from the [DataAbsentReason Code System](http://terminology.hl7.org/CodeSystem/data-absent-reason) following the section on [Missing Data](#missing-data).
 
 ### Mandatory Elements
-Mandatory elements are elements with a minimum cardinality of 1 (min=1). When an element is Mandatory, the data is expected to always be present. Very rarely, it may not be, and in this circumstance the requirements in [Missing Data](#missing-data) **SHALL** be applied.
+Mandatory elements are elements with a minimum cardinality of 1 (min=1). When an element is Mandatory, the data is expected to always be present. Very rarely, it may not be, and in this circumstance the requirements in [Missing Data](#missing-data) **SHALL** be applied. An element can be both _Must Support_ and mandatory, in which case the requirements for mandatory's Missing data requirements apply as described in [Missing Must Support and Mandatory Data](general-requirements.html#missing-must-support-and-mandatory-data).
```
Comment (StephThaoOng) [inline input/pagecontent/general-requirements.md:231]: In [Fixes for FHIR-45065 & FHIR-45102](https://github.com/hl7au/au-fhir-core/pull/167/commits/5cab0ca9b37159205233219da00c984dcb008f8e): Line 231: 

> "An element can be both _Must Support_ and mandatory, in which case the requirements for mandatory’s Missing data requirements apply.."
Consider instead: 
"An element can be both _Must Support_ and mandatory, in which case the missing data requirements for a mandatory element applies.."
to remove the inconsistent capitalisation on "Missing" and perhaps improve readability. This is a suggestion only.

**Comment by dt-r** [comment]
> Fix for FHIR-45159: Line 24: should we update the url link to either:
> 
> 1. https://sparked.csiro.au/index.php/sparked-products-resources/australian-core-data-for-interoperability/
>    or,
> 2. https://sparked.csiro.au/index.php/sparked-products-resources/australian-core-data-for-interoperability/aucdi-release-1/
> 
> Currently the url in the input file re-directs to https://sparked.csiro.au/

When clicked, the built IG lands on: https://sparked.csiro.au/index.php/sparked-products-resources/australian-core-data-for-interoperability/aucdi-release-1/

> In [Fixes for FHIR-45065 & FHIR-45102](https://github.com/hl7au/au-fhir-core/pull/167/commits/5cab0ca9b37159205233219da00c984dcb008f8e): Line 160:
> 
> > Example: ExplanationOfBenefit resource where the patient’s insurance coverage is not available.
> > The text has not been updated to align with the code snippet.
> > Suggest change to something like "Patient resource where the patient's date of birth is not available".

Fixed in the Must Support & Obligation branch.

File: input/pagecontent/general-requirements.md (hunk 1 above)
Comment (dt-r) [reply input/pagecontent/general-requirements.md:231]: Fixes have occurred in the Must Support & Obligation branch.

**Comment by dt-r** [comment]
> Fix for FHIR-45159: Line 24: should we update the url link to either:
> 
> 1. https://sparked.csiro.au/index.php/sparked-products-resources/australian-core-data-for-interoperability/
>    or,
> 2. https://sparked.csiro.au/index.php/sparked-products-resources/australian-core-data-for-interoperability/aucdi-release-1/
> 
> Currently the url in the input file re-directs to https://sparked.csiro.au/

Fixed in master.

**Pull request merged**